libnmstate/netapplier.py

```diff
@@ -128,6 +125,34 @@ def _apply_ifaces_state(
     validator.validate_interfaces_state(desired_state, current_state)
     validator.validate_routes(desired_state, current_state)
 
+
+def _apply_edited_ifaces_without_checkpoint(
+    desired_state, verify_change, commit, rollback_timeout
+):
+    current_state = state.State(netinfo.show())
+
+    _prepare_desired_state_for_apply(current_state, desired_state)
```

Comment:
It would be nicer to return the desired state and assign it to show that it will be changed.

Reply:
Done!

Reply:
It is not done, yet, the code needs to look like `desired_state = _prepare_desired_state_for_apply(current_state, desired_state)` for this.

Reply:
Oops, sorry again. I forgot the assign. Thanks!